R/teal_data-get_code.R

```diff
@@ -108,6 +108,12 @@ setMethod("get_code", signature = "teal_data", definition = function(object, dep
   checkmate::assert_flag(deparse)
 
   code <- if (!is.null(datanames)) {
+    lifecycle::deprecate_warn(
+      when = "0.6.1",
+      what = "teal.data::get_code(datanames)",
+      with = "teal.code::get_code(names)",
+      always = TRUE
+    )
```

Comment:
This is under `if` condition, and not during the opening of the function becuase `datanames` are always passed as they have a default value. So this would trigger the warning always, even though `datanames` would be NULL